fastlane_bot/config/network.py

```diff
@@ -216,7 +216,9 @@ class ConfigNetwork(ConfigBase):
     SOLIDLY_V2_NAME = "solidly_v2"
     VELODROME_V2_NAME = "velodrome_v2"
     SHIBA_V2_NAME = "shiba_v2"
-
+    BUTTER_V3_NAME = "butter_v3"
```

Comment:
I would really like us to stop adding info to the config object(s). Instead, can you move these to the constants.py file? In the *very* near future I'm hoping to dissolve all config objects, so we can make it easier if we stop adding to it asap.

Reply:
Updated this - could you please look again?